source/language/types.rst

```diff
@@ -69,6 +69,15 @@ significantly since there is no need for quantum memory management.
 However, it also means that users or compiler have to explicitly manage
 the quantum memory.
 
+.. code-block:: c
+
+   // Declare a qubit
+   qubit gamma;
+   // Declare a qubit with a Unicode name
+   qubit γ;
+   // Declare a qubit register with 20 qubits
+   qubit[20] qubit_array;
```

Comment:
@blakejohnson Are we allowing this syntax for qubit registers or do we prefer `array[qubit, 20]`?

Reply:
Both are allowed. The types WG will need to discuss whether they are equivalent.